nflow-engine/src/test/java/io/nflow/engine/internal/dao/WorkflowInstanceDaoTest.java

```diff
@@ -147,11 +148,12 @@ public void queryWorkflowInstanceWithMinimalConditions() {
     WorkflowInstance i1 = constructWorkflowInstanceBuilder().build();
     long id = dao.insertWorkflowInstance(i1);
     assertThat(id, not(equalTo(-1)));
-    QueryWorkflowInstances q = new QueryWorkflowInstances.Builder().build();
+    QueryWorkflowInstances q = new QueryWorkflowInstances.Builder().setIncludeCurrentStateVariables(false).build();
```

Comment:
Add #TODO comment to remove setIncludeCurrentStateVariables(false) in 7.0.0 release